cdisutils/storage.py

```diff
@@ -259,8 +259,8 @@ def get_url(self, url):
             raise RuntimeError("{} is not an s3 url".format(url))
         host = parsed_url.netloc
         bucket, key = parsed_url.path.split("/", 2)[1:]
-        bucket = self.get_connection(host).get_bucket(bucket)
-        return bucket.get_key(key)
+        bucket = self.get_connection(host).get_bucket(bucket, validate=False)
+        return bucket.get_key(key, validate=False)
```

Comment:
Does it still work if we keep validation on the key? I think validating the key would be good if it works.